mina.netty/src/main/java/org/kaazing/mina/netty/socket/DatagramChannelIoSessionConfig.java

```diff
@@ -39,10 +39,12 @@ protected final void doSetAll(IoSessionConfigEx config) {
 
             setReceiveBufferSize(datagramConfig.getReceiveBufferSize());
             setSendBufferSize(datagramConfig.getSendBufferSize());
+            /*
             setTrafficClass(datagramConfig.getTrafficClass());
             setBroadcast(datagramConfig.isBroadcast());
             setCloseOnPortUnreachable(datagramConfig.isCloseOnPortUnreachable());
             setReuseAddress(datagramConfig.isReuseAddress());
+            */
```

Comment:
Why is this code still in here but commented out? Either need to explain in a comment or just remove it.